src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -107,7 +110,7 @@ public void parse_compulsoryFieldMissing_failure() {
         assertParseFailure(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + VALID_ADDRESS_BOB
                 + BIRTHDAY_DESC_BOB, expectedMessage);
 
-        // missing address prefix
+        // missing birthday prefix
```

Comment:
Just realised this test and the previous test is exactly the same.

Reply:
LOL, i just realised it too.